vendor/k8s.io/kubernetes/test/utils/crd/crd_util.go

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
These utils are coming from some bigger refactor, but I guess we can just keep them together with this particular commit.